console/src/main/java/org/georchestra/console/dto/AccountImpl.java

```diff
@@ -148,7 +153,7 @@ public String toString() {
                 + ", roomNumber='" + roomNumber + '\'' + ", stateOrProvince='" + stateOrProvince + '\''
                 + ", homePostalAddress='" + homePostalAddress + '\'' + ", shadowExpire='" + shadowExpire + '\''
                 + ", privacyPolicyAgreementDate='" + privacyPolicyAgreementDate + '\'' + ", context='" + context + '\''
-                + ", org='" + org + '\'' + '}';
+                + ", org='" + org + '\'' + ", sshKeys='" + sshKeys + "'}";
```

Comment:
looks like this should be `Arrays.toString(sshKeys)`

Reply:
I thought it would be ok to leave Java do the job, but I think you're right, it would be more explicit to have the toString()

Reply:
oh, the issue is that String[].toString() witll give you garbage